aica_docs/set_up_reinforcement_learning.md

```diff
@@ -211,9 +213,19 @@ class ReachSceneCfg(InteractiveSceneCfg):
         spawn=sim_utils.DomeLightCfg(color=(0.75, 0.75, 0.75), intensity=2500.0),
     )
 ```
+#### Defining the Observations
+
+The environment's **scene** represents its complete state, while **observations** represent the subset of this state 
+visible to the agent. The agent uses these observations to choose its actions.
+
+The `ObservationsCfg` class defines the observations provided to the RL actor as a state vector. This state vector can 
+contain various details, such as joint positions, joint velocities, or the end-effector pose in robotic arms.
 
-The `ObservationsCfg` class specifies the observations passed to the RL actor in the form of a state vector, which can
-include information such as joint positions, joint velocities, and end-effector pose for robotic arms.
+Observations can include multiple terms organized into distinct **observation groups**. Grouping observations in 
+this way allows users to specify separate observation spaces. Each observation term within a single group
+must have the same dimensions.
+
+In the following example, an observation group named `"policy"` is defined. Note that the `"policy"` observation group is required by the RL wrappers.
```

Comment:
Do we really need the quotes in the code block? Feels like the code block is already enough